mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-flex-service-subschema.json

```diff
@@ -39,12 +39,28 @@
                                 "Microsoft.DBforPostgreSQL",
                                 "Microsoft.DBforMariaDB"
                             ]
+                        },
+                        "supportsBatchUpdate": {
+                            "description": "Whether API supports updating multiple config parameters at once",
+                            "type": "boolean"
+                        },
+                        "isFlex": {
+                            "description": "Whether SaaS is an Azure Flexible Server",
+                            "type": "boolean"
+                        },
+                        "apiVersion": {
+                            "description": "Version of Azure REST API, e.g., 2022-01-01-preview",
```

Comment:
Also provide a documentation URL where one could go look these up?
Same for the `provider` param.

Reply:
done